### PR TITLE
(More of) Wide cubestr fix 3v0vx v2

### PR DESCRIPTION
**Pull request by pp-mo**
### Description
Fixes to already-merged #4233 

(previously) ~***NOT*** rebased (yet), in case we wanted to back that out.~

### CODE REVIEW

**Comment by bjlittle** [comment]
@pp-mo What do you still need to do to take this PR out of draft?

File: docs/iris/src/whatsnew/3.0.4.rst [hunk 1]
```diff
@@ -22,6 +22,11 @@ This document explains the changes made to Iris for this release
       :meth:`iris.fileformats.pp.PPDataProxy.__getitem__`. The check for empty slicings
       is no longer needed since (:pull:`4135`) was added. (:pull:`4141`)
```
Comment (bjlittle) [inline docs/iris/src/whatsnew/3.0.4.rst:24]: @pp-mo You still need to remove the above entry, as that was a patch applied in [v3.0.2](https://scitools-iris.readthedocs.io/en/v3.0.3/whatsnew/3.0.2.html)

**Review by bjlittle — CHANGES_REQUESTED** [review]
@pp-mo see the comment for the `whatsnew` change :+1:

File: docs/iris/src/whatsnew/3.0.4.rst (hunk 1 above)
Comment (pp-mo) [reply docs/iris/src/whatsnew/3.0.4.rst:24]: If so, why isn't it already in the whatsnew here under 3.0.2 changes ?

Comment (pp-mo) [reply docs/iris/src/whatsnew/3.0.4.rst:24]: It seems to have vanished between the whatsnew/v3.0.3 and v3.0.4 ...

File: docs/iris/src/whatsnew/3.0.4.rst [hunk 2]
```diff
@@ -82,6 +83,10 @@ This document explains the changes made to Iris for this release
       to avoid the dask 'test access'. This makes loading of netcdf files with a
       large number of variables significantly faster. (:pull:`4135`)
 
+   #. `@pp-mo`_ reverted a change made previously in (:pull:`3659`) to
+      :meth:`iris.fileformats.pp.PPDataProxy.__getitem__`. The check for empty slicings
+      is no longer needed since (:pull:`4135`) was added. (:pull:`4141`)
+
```
Comment (bjlittle) [inline docs/iris/src/whatsnew/3.0.4.rst:89]: @pp-mo Nice, thanks :+1:

**Review by bjlittle — APPROVED** [review]

**Comment by pp-mo** [comment]
Thanks @bjlittle .
Speed service ! 🚀

**Pull request merged**